src/H5C.c

```diff
@@ -348,6 +365,13 @@ H5C_create(size_t max_cache_size, size_t min_clean_size, int max_type_id,
     cache_ptr->il_head = NULL;
     cache_ptr->il_tail = NULL;
 
+    /* Fields supporting VFD SWMR */
```

Comment:
This entire set of initialization here should be replaced by a memset and then setting a few fields to their non-zero values.   We should definitely not be prolonging this terrible code by adding a loop over 4K values to set them individually to NULL.